### ISSUE TITLE
BatchNorm Improvements from "Four Things Everyone Should Know to Improve Batch Normalization"

### ISSUE THREAD

**Issue by michalwols** [michalwols/yann#3]
https://openreview.net/forum?id=HJx8HANFDH
